index.src.html

```diff
@@ -1038,36 +1076,17 @@ <h4 id="algorithm-require-mediation" algorithm>Require User Mediation</h4>
     };
   </pre>
   <dl dfn-for="PasswordCredential">
-    :   <dfn attribute>idName</dfn>
-    ::  This attribute represents the name which will be used for the ID field when submitting the
-        {{PasswordCredential}} to a remote endpoint via {{fetch()}}. It defaults to "`username`",
-        but can be overridden by a developer to match whatever the backend service expects.
-
-    :   <dfn attribute>passwordName</dfn>
-    ::  This attribute represents the name which will be used for the ID field when submitting the
-        {{PasswordCredential}} to a remote endpoint via {{fetch()}}. It defaults to "`password`",
-        but can be overridden by a developer to match whatever the backend service expects.
-
-    :   <dfn attribute>additionalData</dfn>
-    ::  If the developer wishes to specify additional data to insert into the request body when
-        submitting the credential information to a remote endpoint, they can do so by assigning a
-        {{FormData}} or {{URLSearchParams}} object to this attribute. The credential information
-        will be mixed into the object to produce the body. The value is `null` unless otherwise
-        specified.
+    :   <dfn attribute>password</dfn>
+    ::  This attribute represents the password of the credential.
 
     :   {{Credential/[[type]]}}
     ::  The {{PasswordCredential}} [=interface object=] has an internal slot named `[[type]]` whose
-        value is "<dfn const>`password`</dfn>".
+        value is "<dfn const id="password-const">`password`</dfn>".
```

Comment:
You could change this to `for="Credential/[[type]]"` instead. That will give Bikeshed enough of a hint to generate a distinct ID.